CIP-????/README.md

```diff
@@ -0,0 +1,285 @@
+   - Different users can transfer funds with encrypted messages attached between each other, using a preshared passphrase. Only theses users need to know the content. Example: A user buys goods from an online-store, the store provides a preshared-passphrase to the user on their website or via email, the user sends the payment with payment-information encrypted with this passphrase to the store.
+   - Keeping the usecase of a transaction private does not only belong to different entities, but to a single user too. Example: If a user sends funds to a Dex or wants to lend some fund to a friend, he just can add information like 'Sent xxx ADA to bob for xxx' to the outgoing transaction as a documentation using an own choosen private passphrase. This information is stored on the chain and so in the wallet, only the user itself can review the use case of these transactions.
+   - Backwards compatible with CIP-0020
+   - Easy implementation by using well known tools like OpenSSL
+
+## What this CIP is not trying to do
+
+This addition to the original CIP-0020 should not be seen as the end-all-be-all security solution for privacy on the blockchain. There's better options and upcoming Midnight for that. The transaction messages are also not intended to act like chat messages on the chain.
+
+&nbsp;<p>
+
+# Specification - Encrypted message
+
+The specification update for encrypted messages takes advantage of the simple original design, which is leaving room for additional json-keys not affecting the parsing of the content at all. The only outcome if a receiver does not process the encrypted content is, that the encrypted message is shown instead of an maybe autodecrypted one. But even the encrypted base64 strings fit into the max. 64char long string restriction. So it does not break any tools. More on the autodecryption later. 
+
+## Format:
+``` 
+{ 
+  "674":
+         { 
+           "enc": "<encryption-method>",
+           "msg": 
+                  [ 
+                    "base64-string 1", "base64-string 2", "base64-string 3" ...
+                  ]
+         }
+}
+```
+The format is identical to the original one, with a simple addition of the `enc` (encryptionmode) entry.
+
+The value given in the `enc` field references the type of encryption is used. Starting with a simple implementation named `basic`. There is room to add additional encryption method in the future like using ChaCha20/Poly1305 or using public/private key encryption. Also there is the possibility to not encode the metadata in the standard JSON format, but using CBOR encoding instead.
```

Comment:
> Also there is the possibility to not encode the metadata in the standard JSON format, but using CBOR encoding instead.

I am not sure to follow this sentence. While metadata are typically (and wrongly IMO) represented as JSON, they always end up encoded as CBOR objects on-chain. While there's a direct -- and relatively obvious -- mapping from JSON to CBOR, it usually work out okay. 

For the sake of clarify though, we typically prefer that on-chain format comes with an unambiguous .cddl definition (preferably in an annex standalone file).

Reply:
I have added a [cddl](./format.cddl) description, please let me know if it is sufficient.